extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/sql_load_scirpt/ImportSqlLoadScriptTestCase.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.hibernate.orm.sql_load_scirpt;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class ImportSqlLoadScriptTestCase {
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addAsResource("application-import-load-script-test.properties", "application.properties")
+                    .addAsResource("import.sql"));
+
+    @Test
+    public void testImportSqlLoadScriptTest() {
+    }
```

Comment:
some here. How can we make sure that:

- it actually started Hibernate ORM successfully
- it didn't use the default configuration
- the specified `import.sql` script was run

Reply:
- The fact that `QuarkusUnitTest` runner did not throw was enough to make sure that the load-script did not cause the crash. Probably we can add something here? 

For second and last points, this test need to be enriched with verification that there is something loaded in the database: I'll add this part later on this evening.

Reply:
The same verifications can be added when loading a different sql script: `SqlLoadScriptPresentTestCase `.

Reply:
Done.